### PR TITLE
[full-ci] Rename Permissions

### PR DESCRIPTION
**Pull request by kobergj**
Renames permissions

ATTENTION: DISABLES E2E TESTS

### CODE REVIEW

**Comment by kobergj** [comment]
@kulmann  @micbar as discussed draft for new permission names. Please double check. I will implement reva part once we agreed on permission names

**Comment by micbar** [comment]
@kobergj @kulmann I would not include the scope in the permission name.

**Comment by kobergj** [comment]
@micbar not sure what you mean. Could you make a suggestion on one permission?

File: services/settings/pkg/store/defaults/defaults.go [hunk 1]
```diff
@@ -21,17 +21,22 @@ const (
...
-	SettingsManagementPermissionName string = "settings-management"
+	SettingsManagementPermissionName string = "Settings.ReadWrite"
 
-	// LanguageReadWriteID is the hardcoded setting UUID for the language read write permission
+	// LanguageReadWriteID is the hardcoded setting UUID for the language read write all permission
 	LanguageReadWriteID string = "7d81f103-0488-4853-bce5-98dcce36d649"
-	// LanguageReadWriteName is the hardcoded setting name for the language read write permission
-	LanguageReadWriteName string = "language-readwrite"
+	// LanguageReadWriteName is the hardcoded setting name for the language read write all permission
+	LanguageReadWriteName string = "Language.ReadWrite.All"
```
Comment (micbar) [inline services/settings/pkg/store/defaults/defaults.go:34]:
```suggestion
	LanguageReadWriteName string = "Language.ReadWrite"
```

File: services/settings/pkg/store/defaults/defaults.go [hunk 2]
```diff
@@ -21,17 +21,22 @@ const (
...
+
+	// LanguageReadWriteSelfID is the hardcoded setting UUID for the language read write self permission
```
Comment (micbar) [inline services/settings/pkg/store/defaults/defaults.go:36]: We do not need that second scope for the Language permisson. We always append the scope to the list permissions response.

## Example

```json
{
    "permissions":[
        "role-management.all",
        "settings-management.all",
        "language-readwrite.all",
        "Drive.ReadWrite.all",
        "Drive.ReadWriteEnabled.all",
        "change-logo.all",
        "create-space.all",
        "list-all-spaces.all",
        "delete-all-home-spaces.all",
        "account-management.all",
        "group-management.all",
        "delete-all-spaces.all",
        "PublicLink.Write.all",
        "set-space-quota.all",
        "Drive.ReadWriteQuota.Project.all"
    ]
}
```

Comment (kobergj) [reply services/settings/pkg/store/defaults/defaults.go:36]: :sob: We need to change it. It makes no sense on one half of the permissions and is wrong on the other half

Comment (kobergj) [reply services/settings/pkg/store/defaults/defaults.go]: Removed all constraints in names and double checked explicit constraints are correct. Please check again

**Comment by ownclouders** [comment]
:boom: Acceptance test [<strong>cs3ApiTests-ocis</strong>](https://drone.owncloud.com/owncloud/ocis/22718/46/1) failed. Further test are cancelled...

File: services/settings/pkg/store/defaults/defaults.go (hunk 2 above)
Comment (micbar) [reply services/settings/pkg/store/defaults/defaults.go]: I think we are not aligned. Every permission has a scope. Example https://github.com/owncloud/ocis/blob/master/services/settings/pkg/store/defaults/defaults.go#L146

**Review by micbar — APPROVED** [review]
From the permissions wording it looks fine now.

**Comment by kobergj** [comment]
e2e tests fail as expected. Rest is green :+1: PRs will wait until web is ready so we can minimize time without e2e tests @kulmann could you ping me when we can start the merge-orgy? :smile:

**Comment by kulmann** [comment]
> e2e tests fail as expected. Rest is green 👍 PRs will wait until web is ready so we can minimize time without e2e tests @kulmann could you ping me when we can start the merge-orgy? 😄

ping. Web is ready (see PR from @JammingBen https://github.com/owncloud/web/pull/9138)

Just to have it written down somewhere: we will now:
- disable e2e tests in this PR and then merge it
- bump the ocis commid id in the drone env file in https://github.com/owncloud/web/pull/9138 and merge it into `stable-7.0`
- create a pre-release in web
- bump the web pre-release in ocis and enable the e2e tests again in the same PR

correct? any additions?

**Pull request merged**